INIT: Using found mnist data
INIT: Finished creating train dataset
Traceback (most recent call last):
  File "RunTest.py", line 192, in <module>
    main(args)
  File "RunTest.py", line 114, in main
    for X,Y in train_data_gen[:3]:
  File "/com.docker.devenvironments.code/project/large_models/datagen.py", line 69, in __getitem__
    batches = self.df[index * self.batch_size:(index+1) * self.batch_size]
TypeError: unsupported operand type(s) for *: 'slice' and 'int'